netconf/transport.go

```diff
@@ -20,6 +20,10 @@ const (
 	msgSeperator_v11 = "\n##\n"
 )
 
+const (
+	BrokenConnectionError = "broken connection during write"
```

Comment:
Sentinal errors should be defined globally and not by a string.  

This should be
```
var BrokenConnectionErr = errors.New("broken connection during write")
```
```

Reply:
Ok, sounds good

Do we need to know the writelen to be better know what kind of error?
I mean, does EOF give us enough context?

Reply:
In this case probably.  We could wrap the error but that is supported in Go 1.15 and beyond and this package should strive to support the previous two versions.

Reply:
:+1: